index.js

```diff
@@ -89,4 +97,25 @@ module.exports = function (processors, options) {
   }
 
   return stream
+})
+
+
+function withConfigLoader(cb) {
+  return function (plugins, options) {
+    if (typeof plugins === 'undefined') {
+      return cb(postcssLoadConfig)
```

Comment:
@michael-ciniawsky Ok, I've added a way to specify the config location. This seems to be ready to be merged. It would be nice to actually test it on the real project because I'm not using gulp anymore.